client/setup.py

```diff
@@ -24,7 +24,16 @@
     install_requires=["SQLAlchemy", "alembic", "securedrop-sdk", "python-dateutil", "arrow"],
     python_requires=">=3.5",
     url="https://github.com/freedomofpress/securedrop-client",
-    packages=["securedrop_client", "securedrop_client.gui", "securedrop_client.resources"],
+    packages=setuptools.find_packages(include=["securedrop_client", "securedrop_client.*"]),
+    package_data={
```

Comment:
I think we can skip this part if we keep the MANIFEST.in? (Also not all the CSS is in resources/, we'd need some `**/*.css` I think.